develop/develop/urls.py

```diff
@@ -23,4 +23,5 @@
     path('sales/manage/', include('vendor.urls.vendor_admin')),
     path('admin/', admin.site.urls),
     path('accounts/', include('allauth.urls')),
+    path('siteconfigs/', include('siteconfigs.urls')),
```

Comment:
very minor, but i don't think there's any siteconfigs urls that are needed at the moment. it's fine to leave in if you prefer though